packages/flatseal.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Flatseal < Package
+  description 'Flatseal is a graphical utility to review and modify permissions from your Flatpak applications.'
+  homepage 'https://github.com/tchx84/Flatseal'
+  version '1.6.8'
+  license 'GPL-3+'
+  compatibility 'all'
+  source_url 'https://github.com/tchx84/Flatseal/archive/v1.6.8.tar.gz'
+  source_sha256 '28d05995effa2858483283cbc9fb54c55bf4dcb6267e4d7d495a32ff724806b1'
+
+  depends_on 'libhandy'
+  depends_on 'desktop_file_utils'
+  depends_on 'gjs'
+  depends_on 'gtk3'
```

Comment:
Do we need `depends_on 'samurai' => :build`?

Reply:
Samurai should be a dependency of meson since we're transitioning from ninja to samurai.

Reply:
But that currently isn't the way we have it configured.

Reply:
Fixed!

Reply:
What was fixed?

Reply:
8d4a3a7

Reply:
Oh, I see you added samurai to meson below.